docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+if [[ ! -d "$DATA_DIR" ]]; then
+        exit 1
+fi
+
+./manage.py initialize
```

Comment:
This command only runs [`migrate`](https://github.com/liquidinvestigations/core/blob/6c6ccc7f520cba5575794652df36165e4aba7102/liquidcore/home/management/commands/initialize.py#L10) (it used to do a lot more) - so you can remove that file and replace this line with `./manage.py migrate`.